gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1030,17 +1097,24 @@ def format_validated_ht_for_export(
     ht = ht.transmute(info=hl.struct(**info_annot_mapping))
 
     logger.info("Adjusting VCF incompatible types...")
-    # Reformat AS_SB_TABLE for use in adjust_vcf_incompatible_types
-    ht = ht.annotate(
-        info=ht.info.annotate(
-            AS_SB_TABLE=hl.array([ht.info.AS_SB_TABLE[:2], ht.info.AS_SB_TABLE[2:]])
+    if data_type != "joint":
+        # Reformat AS_SB_TABLE for use in adjust_vcf_incompatible_types
+        ht = ht.annotate(
+            info=ht.info.annotate(
+                AS_SB_TABLE=hl.array([ht.info.AS_SB_TABLE[:2], ht.info.AS_SB_TABLE[2:]])
+            )
         )
-    )
     # The Table is already split so there are no annotations that need to be
     # pipe delimited.
     ht = adjust_vcf_incompatible_types(ht, pipe_delimited_annotations=[])
 
     logger.info("Rearranging fields to desired order...")
+    if data_type == "joint":
+        vcf_info_reorder = [
+            f"{f}_{dt}"
+            for dt in ["joint", "exomes", "genomes"]
+            for f in vcf_info_reorder
+        ]
```

Comment:
I think we should move `exomes_filters` and `genomes_filters` up in the order as well. Could you extend this list to include them. Right now they are scattered in the fields.

Reply:
Do you want these two before everything else?

Reply:
Please put them both at the end of the list